quiz.css

```diff
@@ -0,0 +1,60 @@
+* {
```

Comment:
Things like ```font-family``` are OK to put on the ```body``` tag, as any and all text will end up in the body, and it's more specific than the everything ```*```